lib/cc/services/github_pull_requests.rb

```diff
@@ -104,6 +107,9 @@ def update_status(state, description)
         target_url:  @payload["details_url"],
         context:     "codeclimate"
       }
+      if state == "error"
+        params.delete(:target_url)
```

Comment:
@pbrisbin @brynary Hmm. It seems to me that right now, the `codeclimate-services` entity is not just getting a payload and relaying it, but making some additional calculations - e.g. formatting messages based on number of issues fixed, adding a `context: 'codeclimate'` to params passed to http request. 

However, we recently updated finalizer to also do some analysis, including checking whether errored state is due to worker_version incompatibility and adding a unique message to payload, if that's the case.

On one hand, it seems nice for `finalizer` (or whatever other entity) to just be able to send a bunch of information to `codeclimate-services` github_PR and have cc-services know what to do with it.

On the other hand, it seems nice to have cc-services not know too much about how the stuff it's handling gets calculated - it just knows it needs to send message x to place y. 

So, yeah. Agree. Ultimately I favor minimizing analysis in cc-services, which would mean not having it read `state` to decide whether to add `details_url` or not.